frontend/views/containers/dashboard/GroupMembersActivity.vue

```diff
@@ -139,20 +147,20 @@ export default ({
       }
 
       return {
-        'onTimePayments': this.onTimePayments.length <= 1
-          ? L('{span_}{membercount} member{_span} have {strong_} on-time payment streaks{_strong}', argsMap['onTimePayments'])
+        'onTimePayments': this.onTimePayments.length === 1
+          ? L('{span_}{membercount} member{_span} has {strong_} on-time payment streaks{_strong}', argsMap['onTimePayments'])
           : L('{span_}{membercount} members{_span} have {strong_} on-time payment streaks{_strong}', argsMap['onTimePayments']),
-        'haventLoggedIn': this.haventLoggedIn.length <= 1
-          ? L('{span_}{membercount} member{_span} haven\'t {strong_} logged in past {days} days or more {_strong}', argsMap['haventLoggedIn'])
+        'haventLoggedIn': this.haventLoggedIn.length === 1
+          ? L('{span_}{membercount} member{_span} hasn\'t {strong_} logged in past {days} days or more {_strong}', argsMap['haventLoggedIn'])
           : L('{span_}{membercount} members{_span} haven\'t {strong_} logged in past {days} days or more {_strong}', argsMap['haventLoggedIn']),
-        'noIncomeDetails': this.noIncomeDetails.length <= 1
-          ? L('{span_}{membercount} member{_span} haven\'t {strong_} entered income details{_strong}', argsMap['noIncomeDetails'])
+        'noIncomeDetails': this.noIncomeDetails.length === 1
+          ? L('{span_}{membercount} member{_span} hasn\'t {strong_} entered income details{_strong}', argsMap['noIncomeDetails'])
           : L('{span_}{membercount} members{_span} haven\'t {strong_} entered income details{_strong}', argsMap['noIncomeDetails']),
-        'missedPayments': this.missedPayments.length <= 1
-          ? L('{span_}{membercount} member{_span} have {strong_} missed payments {_strong}', argsMap['missedPayments'])
+        'missedPayments': this.missedPayments.length === 1
+          ? L('{span_}{membercount} member{_span} has {strong_} missed payments {_strong}', argsMap['missedPayments'])
           : L('{span_}{membercount} members{_span} have {strong_} missed payments {_strong}', argsMap['missedPayments']),
-        'noVotes': this.noVotes.length <= 1
-          ? L('{span_}{membercount} member{_span} haven\'t {strong_} voted in the last {proposalcount} proposals {_strong}', argsMap['noVotes'])
+        'noVotes': this.noVotes.length === 1
+          ? L('{span_}{membercount} member{_span} hsn\'t {strong_} voted in the last {proposalcount} proposals {_strong}', argsMap['noVotes'])
```

Comment:
typo: `hsn`

Also, in all of the above where you have `this.haventLoggedIn.length === 1`, please remove `{membercount}` and replace it with `1`, so that it's clear to the translator that this is a version of the sentence that is singular and therefore doesn't require the variable, because it's always 1